Chrysalis-Papilio/Soen-490-Papilio
# PAPILIO-23-Activity-Creation-UI

# General info
Implemented static UI for user activity creation

**Issue number**: 
PAPILIO-23

**Task description**: 
 - Static UI for the user to create their own activity

# Testing
To test this feature you have to:

1. Sign in
2. Go to 'My Activities' tab
3. Click on '+ Create Activity'
4. Ensure the layout  matches UI mockups
5. Ensure text fields are editable
6. Tap back arrow at top of page to bring you back to the 'My Activities' tab

# Notes
- The 'IMPORT PICTURE' and '+ CREATE ACTIVITY' buttons found at the bottom of the page are non-functional
- UI tests will be added shortly
